packages/js/product-editor/src/components/iframe-editor/utils/tests/are-blocks-empty.test.ts

```diff
@@ -0,0 +1,199 @@
+/**
+ * Internal dependencies
+ */
+import { areBlocksEmpty } from '../are-blocks-empty';
+
+describe( 'areBlocksEmpty', () => {
+	it( 'should return false if there is more than one block', () => {
+		const blocks = [
+			{
+				name: 'core/paragraph',
+				attributes: [],
+				innerBlocks: [],
+				isValid: true,
+				clientId: '1',
+			},
+			{
+				name: 'core/paragraph',
+				attributes: [],
+				innerBlocks: [],
+				isValid: true,
+				clientId: '2',
+			},
+		];
+
+		expect( areBlocksEmpty( blocks ) ).toBe( false );
+	} );
+
+	it( 'should return false if there is one block that is not a paragraph block', () => {
+		const blocks = [
+			{
+				name: 'core/heading',
+				attributes: [],
+				innerBlocks: [],
+				isValid: true,
+				clientId: '1',
+			},
+		];
+
+		expect( areBlocksEmpty( blocks ) ).toBe( false );
+	} );
+
+	it( 'should return false if the paragraph block has content', () => {
+		const blocks = [
+			{
+				name: 'core/paragraph',
+				attributes: {
+					content: 'Some content',
+				},
+				innerBlocks: [],
+				isValid: true,
+				clientId: '1',
+			},
+		];
+
+		expect( areBlocksEmpty( blocks ) ).toBe( false );
+	} );
+
+	it( 'should return false if the paragraph block has no content but a backgroundColor', () => {
+		const blocks = [
+			{
+				name: 'core/paragraph',
+				attributes: {
+					content: '',
+					backgroundColor: 'red',
+				},
+				innerBlocks: [],
+				isValid: true,
+				clientId: '1',
+			},
+		];
+
+		expect( areBlocksEmpty( blocks ) ).toBe( false );
+	} );
+
+	it( 'should return false if the paragraph block has no content but any attribute', () => {
+		const blocks = [
+			{
+				name: 'core/paragraph',
+				attributes: {
+					foo: 'bar',
+				},
+				innerBlocks: [],
+				isValid: true,
+				clientId: '1',
+			},
+		];
+
+		expect( areBlocksEmpty( blocks ) ).toBe( false );
+	} );
+
+	it( 'should return true if an empty block list is passed', () => {
```

Comment:
Referring to the previous comment, this unit test seems to prove what I mentioned.